Electron/src/__tests__/components/Sidebar/AddSongPlayListAccordion.test.tsx

```diff
@@ -207,9 +212,30 @@ test('AddSongPlaylistAccordion submit song correct', async () => {
       target: { value: 'Rock' },
     });
     fireEvent.change(dropdown, { target: { value: 'Rock' } });
-    fireEvent.change(fileInputElement, { target: { files: [file] } });
   });
 
+  // Valid files
```

Comment:
Should we have different tests for each case (mp3,flac,wav) ? Example: test_flac, test_invalid_file etc?

Reply:
Sorry, but I don't understand this.